schemas/targets.schema.json

```diff
@@ -0,0 +1,157 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "signatures": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "keyid": {
```

Comment:
This is a valid assumption [for now](https://theupdateframework.github.io/specification/v1.0.30/index.html#role-keyid), but note that it will change in the [future](https://github.com/theupdateframework/taps/blob/master/tap12.md).